libraries/fc/src/log/logger_config.cpp

```diff
@@ -76,6 +96,9 @@ namespace fc {
          }
          auto ap = fact_itr->second->create( cfg.appenders[i].args );
          log_config::get().appender_map[cfg.appenders[i].name] = ap;
+         if( cfg.appenders[i].type == "zipkin" ) {
+            zipkin_appender = ap;
```

Comment:
What if there's more than one zipkin appender?

Reply:
Reworked this to look at type directly.